sklearn/ensemble/_hist_gradient_boosting/_predictor.pyx

```diff
@@ -16,15 +16,7 @@ from .types cimport X_BINNED_DTYPE_C
 from .types cimport node_struct
 
 
-def _predict_from_numeric_data(nodes, numeric_data, out):
-    _predict_from_numeric_data_parallel(nodes, numeric_data, out)
-
-
-def _predict_from_binned_data(nodes, binned_data, out):
-    _predict_from_binned_data_parallel(nodes, binned_data, out)
-
-
-cdef void _predict_from_numeric_data_parallel(
+def _predict_from_numeric_data_parallel(
```

Comment:
I'd just call it `_predict_from_numeric_data` (same for the binned version) but I don't mind